app/views/bus_stops/by_route.haml

```diff
@@ -3,6 +3,8 @@
 - if @stops.empty?
   No stops on this route
 - @stops.each do |stop|
-  = link_to edit_bus_stop_path(stop.hastus_id) do
-    %button.btn.btn-default= stop.name
+  %ul
```

Comment:
A ul per stop isn't right. Should be one ul and a li per stop.

Reply:
oopsies